extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -705,10 +713,7 @@ export class AmpA4A extends AMP.BaseElement {
           // Need to know if creative was verified as part of render outside
           // viewport but cannot wait on promise.  Sadly, need a state a
           // variable.
-          this.isVerifiedAmpCreative_ = !!creative;
-          // TODO(levitzky) If creative comes back null, we should consider re-
-          // fetching the signing server public keys and try the verification
-          // step again.
+          this.isVerifiedAmpCreative_ = !this.isFluid && !!creative;
```

Comment:
Is this TODO addressed, or just no longer needed/accurate?

Reply:
I don't think this is relevant anymore given all the recent changes to the signing / verifying process.

Reply:
Thanks for clarifying.